graph/patterns/default-properties.md

```diff
@@ -0,0 +1,201 @@
+# Default properties
+
+Microsoft Graph API Design Pattern
+
+*The default properties pattern allows API producers to omit specific properties from the response unless they are explicitly requested using `$select`. The pattern also uses an instance annotation to inform callers that other properties are also available. The same annotation is also use to encourage callers to use `$select`.*
+
+## Problem
+
+API producers want to control the set of properties that their entities return by default, when the incoming request does not specify a `$select`. This can be desirable when an entity type has many properties or an API producer needs to add properties that are computationally expensive to return by default.
+
+## Solution
+
+For incoming requests targeting an entity type where the caller does not specify a `$select`` clause, API producers **may** return a subset of the entity type's properties, omitting computationally expensive properties.
```

Comment:
Is this pattern limited to entity types?  Or can we use the same pattern for complex types?  e.g. A function that returns a collection of complex types?

Reply:
Sure, it could apply to complex types.  We only implemented it so far for entity types.

Reply:
Triage: Doc what is implemented and file a work item to extend to complex types

Reply:
Added a work item for this.